core/arch/arm/plat-rcar/link.mk

```diff
@@ -1,3 +1,5 @@
 include core/arch/arm/kernel/link.mk
 
+SRECFLAGS = --srec-forceS3 --adjust-vma=$(CFG_TZDRAM_START)
```

Comment:
`?=` perhaps, in case one would want to set this from the environment? (that is, support`SRECFLAGS=foo make` or `export SRECFLAGS=foo; make` in addition to `make SRECFLAGS=foo`).